onnxruntime/core/providers/nuphar/scripts/symbolic_shape_infer.py

```diff
@@ -611,15 +611,25 @@ def _infer_OneHot(self, node):
         vi.CopyFrom(helper.make_tensor_value_info(node.output[0], self.known_vi_[node.input[2]].type.tensor_type.elem_type, new_shape))
 
     def _infer_Pad(self, node):
+        if get_opset(self.out_mp_) <= 10:
+            pads = get_attribute(node, 'pads')
+        else:
+            pads = self._try_get_value(node, 1)
```

Comment:
>_try_get_value [](start = 24, length = 14)

pads seems not optional, so should use get_value instead of try_get_value

Reply:
Fixed. Thanks.